src/intervals/functions.jl

```diff
@@ -233,6 +233,15 @@ function sqrt(a::Interval{T}) where T
     @round(sqrt(a.lo), sqrt(a.hi))  # `sqrt` is correctly-rounded
 end
 
+
+
+function cbrt(a::Interval{T}) where T
+    isempty(a) && return a
+
+    Interval(cbrt(a.lo), cbrt(a.hi)) 
```

Comment:
This does not take account of directed rounding.

Reply:
e.g. try `cbrt(1..3)` instead and compare to the result when you use a BigFloat interval.

Reply:
We will need a similar function like sqrt_round in [FastRounding.jl](https://github.com/JeffreySarnoff/FastRounding.jl/blob/master/src/FastRounding.jl), right?

Reply:
I think you can mimic the code for `sqrt`. Have you tried that?